[2023-09-11T08:12:02.724+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mlb_parser.data_clean manual__2023-09-11T08:11:55.185910+00:00 [queued]>
[2023-09-11T08:12:02.730+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mlb_parser.data_clean manual__2023-09-11T08:11:55.185910+00:00 [queued]>
[2023-09-11T08:12:02.731+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-11T08:12:02.738+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): data_clean> on 2023-09-11 08:11:55.185910+00:00
[2023-09-11T08:12:02.743+0000] {standard_task_runner.py:57} INFO - Started process 24697 to run task
[2023-09-11T08:12:02.745+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mlb_parser', 'data_clean', 'manual__2023-09-11T08:11:55.185910+00:00', '--job-id', '381', '--raw', '--subdir', 'DAGS_FOLDER/mlb_parser.py', '--cfg-path', '/tmp/tmpqs4_kv1g']
[2023-09-11T08:12:02.748+0000] {standard_task_runner.py:85} INFO - Job 381: Subtask data_clean
[2023-09-11T08:12:02.775+0000] {task_command.py:415} INFO - Running <TaskInstance: mlb_parser.data_clean manual__2023-09-11T08:11:55.185910+00:00 [running]> on host 2e8147b3a465
[2023-09-11T08:12:02.819+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andy_hsu' AIRFLOW_CTX_DAG_ID='mlb_parser' AIRFLOW_CTX_TASK_ID='data_clean' AIRFLOW_CTX_EXECUTION_DATE='2023-09-11T08:11:55.185910+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-11T08:11:55.185910+00:00'
[2023-09-11T08:12:02.827+0000] {logging_mixin.py:151} INFO - ['nationals', 'braves', 'mets', 'marlins', 'phillies']
[2023-09-11T08:12:02.829+0000] {logging_mixin.py:151} INFO - https://www.mlb.com/nationals/stats/
[2023-09-11T08:12:07.057+0000] {logging_mixin.py:151} INFO - https://www.mlb.com/braves/stats/
[2023-09-11T08:12:09.302+0000] {logging_mixin.py:151} INFO - https://www.mlb.com/mets/stats/
[2023-09-11T08:12:11.269+0000] {logging_mixin.py:151} INFO - https://www.mlb.com/marlins/stats/
[2023-09-11T08:12:13.173+0000] {logging_mixin.py:151} INFO - https://www.mlb.com/phillies/stats/
[2023-09-11T08:12:15.788+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/mlb_parser.py", line 135, in main
    all_team.to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 2878, in to_sql
    return sql.to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 768, in to_sql
    with pandasSQL_builder(con, schema=schema, need_transaction=True) as pandas_sql:
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 832, in pandasSQL_builder
    return SQLDatabase(con, schema, need_transaction)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1536, in __init__
    con = create_engine(con)
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/deprecations.py", line 375, in warned
    return fn(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 514, in create_engine
    u = _url.make_url(url)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 738, in make_url
    return _parse_url(name_or_url)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 799, in _parse_url
    raise exc.ArgumentError(
sqlalchemy.exc.ArgumentError: Could not parse SQLAlchemy URL from string 'postgres_mlb_parser'
[2023-09-11T08:12:15.802+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=mlb_parser, task_id=data_clean, execution_date=20230911T081155, start_date=20230911T081202, end_date=20230911T081215
[2023-09-11T08:12:15.810+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 381 for task data_clean (Could not parse SQLAlchemy URL from string 'postgres_mlb_parser'; 24697)
[2023-09-11T08:12:15.842+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-11T08:12:15.851+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
